recipes-dom0/dom0-image-thin-initramfs/dom0-image-thin-initramfs.bbappend

```diff
@@ -48,3 +48,8 @@ python do_configure_append() {
     bb.build.exec_func("add_to_local_conf", d)
 }
 
+populate_vmlinux () {
+    find ${STAGING_KERNEL_BUILDDIR} -iname "vmlinux*" -exec mv {} ${DEPLOY_DIR_IMAGE} \;
```

Comment:
Do you really want to move the file? May it is better to copy it?

Reply:
It is just an archived copy of the source, so we need it only in the deploy directory